templates/google-native/app/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+    application
+    kotlin("jvm") version "1.9.10"
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("org.virtuslab:pulumi-kotlin:0.9.4.0")
+    implementation("org.virtuslab:pulumi-google-native-kotlin:0.31.1.1")
```

Comment:
Does this work? In the E2E tests I encountered an issue where I has to exclude some stuff due to a versioning conflict
https://github.com/VirtuslabRnD/pulumi-kotlin/blob/f01ac54a81a0609701ef72ab70b0c4b20458612e/examples/google-native-sample-project/pom.xml#L21-L43

Reply:
Btw, aside from the potential dependency conflict, it shouldn't be necessary to include the core SDK explicitly, just the Google Native dependency should be fine.